scripts/platform.sh

```diff
@@ -7,9 +7,6 @@ rm -rf platform lib/src/platform
 git clone https://github.com/opentdf/platform.git
 
 # Generate Typescript code
-cmd=$(npx buf --version)
-if [ $? -ne 0 ]; then
-  npm install @bufbuild/protoc-gen-es @bufbuild/buf @connectrpc/protoc-gen-connect-es
```

Comment:
I've put this dependencies as `devDependencies` in the package.json. They do no need to be installed here